docs/development/howto.rst

```diff
@@ -224,6 +224,12 @@ repository. It is recommended that developers have `$GAMMAPY_DATA` environment v
 where they have fetched the `gammapy-data <https://github.com/gammapy/gammapy-data>`__  Github repository,
 so they can push and pull eventual modification of its content.
 
+Making a pull request which skips GitHub Actions
```

Comment:
Actually you can skip CI for a specific commit. So it does not specifically applies to a PR in general but to any commit that is incomplete or draft.
Maybe you can be more general here.